ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+### Issue
+
+- [ ] This is a bug report.
+- [ ] This is a feature request.
```

Comment:
Can we please use labels like https://github.com/nasa-gcn/.github/labels/bug and https://github.com/nasa-gcn/.github/labels/enhancement instead of checkboxes?

Reply:
Good call! I also thought that coupling the task list feature of gh markdown was not the right tool for the job here.